app/components/card_component.html.erb

```diff
@@ -1,8 +1,22 @@
-<div class="card">
+<div class="card<%= " card--no-border" unless border %>">
```

Comment:
I think this can be simplified a bit by introducing a `card_classes` method on the component that builds an array

```erb
<%= tag.div("card", class: card_classes) do %>
...
```

Reply:
Sure, I'll take a look